build_data.js

```diff
@@ -771,14 +772,14 @@ function translationsToYAML(translations) {
 }
 
 
-function writeRapidConfig() {
-    var readRapidConfig = readFileProm('data/rapid_config.yaml', 'utf8'); 
+function writeRapidConfig(build_type) {
+  var readRapidConfig = readFileProm('data/rapid_config.yaml', 'utf8'); 
 
-    return Promise.all([readRapidConfig]).then(function(data) {
-        var config = YAML.load(data[0]); 
+  return Promise.all([readRapidConfig]).then(function(data) {
+      var config = YAML.load(data[0]); 
 
-        return writeFileProm('data/rapid_config.json', JSON.stringify(config, null, 4)); 
-    }); 
+      return writeFileProm('data/rapid_config.json', JSON.stringify(config[build_type], null, 4)); 
```

Comment:
Depending on the build type, we grab a different config block to place into the config.json. This means that the rapid runtime code does not need to decide which config block to evaluate- it only ever has the one that we switch in & out at build time.